deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -27,3 +30,38 @@ def callback_integrity_message(line):
         match = re.match(r"(\d{4}/\d{2}/\d{2} \d{2}:\d{2}:\d{2}).*({.*?})$", line)
         if match:
             return datetime.strptime(match.group(1), '%Y/%m/%d %H:%M:%S'), json.dumps(match.group(2))
+
+
+def callback_detect_event(line):
+    """
+    Detect an 'event' type FIM log.
+    """

```

Comment:
Please, add Args in the documentation.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4181/commits/3cbb536410bd51c5fd944a554e40c3750c0274f4